.env.devnet

```diff
@@ -0,0 +1,105 @@
+# Deployer private key
+PRIVATE_KEY=0x
+
+# Network to deploy to (mainnet, testnet, or devnet)
+NETWORK="devnet"
+
+# Owner address for L1LiskToken contract
+L1_TOKEN_OWNER_ADDRESS=0x02cBF39018B40a48f0d95d3a0b2eeeF3D18e42Fe
```

Comment:
This address is not controlled by users who want to locally test any deployment.
Not sure if we want to set any address here and below.